apis/config/v1alpha1/cluster_colocation_profile_types.go

```diff
@@ -26,19 +26,17 @@ import (
 
 // ClusterColocationProfileSpec is a description of a ClusterColocationProfile.
 type ClusterColocationProfileSpec struct {
-	// NamespaceSelector decides whether to mutate/validate Pods on an object based
-	// on whether the namespace for that object matches the selector.
+	// NamespaceSelector decides whether to mutate/validate Pods based
+	// on whether the namespace for that matches the selector.
 	// Default to the empty LabelSelector, which matches everything.
 	// +optional
 	NamespaceSelector *metav1.LabelSelector `json:"namespaceSelector,omitempty"`
 
-	// ObjectSelector decides whether to mutate/validate Pods based on if the
+	// Selector decides whether to mutate/validate Pods based on if the
```

Comment:
Selector decides whether to mutate/validate Pods if the Pod has matching labels.